flatt.js

```diff
@@ -0,0 +1 @@
+const flatt = arrays => arrays.reduce((prev, next) => prev.concat(next))
```

Comment:
it's a good thing to supply an initial value for `reduce`, it could be an empty array here :)